backend/api/serializers/FuelCode.py

```diff
@@ -100,6 +103,52 @@ class FuelCodeSaveSerializer(serializers.ModelSerializer):
         queryset=TransportMode.objects.all()
     )
 
+    def validate(self, data):
+        print('wow')
+        # check if fuel code is correct
+        fuel_code = data.get('fuel_code')
+        fuel_code_version = data.get('fuel_code_version')
+        fuel_code_version_minor = data.get('fuel_code_version_minor')
+
+        matches = FuelCode.objects.filter(
+            fuel_code=fuel_code,
+            fuel_code_version=fuel_code_version,
+            fuel_code_version_minor=fuel_code_version_minor
+        )
+
+        print(fuel_code)
```

Comment:
For debugging?

Reply:
whoops. thanks for catching this. ill get rid of it while i fix the merge conflict